src/shared/components/Grid/Grid.tsx

```diff
@@ -61,6 +48,27 @@ const Grid: React.FC<GridProps> = ({
   )
 }
 
+type ContainerProps = Required<Pick<GridProps, 'gap' | 'maxColumns' | 'minWidth' | 'repeat'>>
+
+const maxColumnsCss = ({ maxColumns }: ContainerProps) =>
```

Comment:
Why is this needed exactly?

Reply:
So that we don't have to use `maxColumns` every time, for channel previews, which don't fill available space, max columns don't make much sense